hivemind/averaging/matchmaking.py

```diff
@@ -511,7 +512,7 @@ async def _update_queue_periodically(self, key_manager: GroupKeyManager) -> None
             self.update_finished.set()
 
             await asyncio.wait(
-                {self.running.wait(), self.update_triggered.wait()},
+                set(map(asyncio.create_task, (self.running.wait(), self.update_triggered.wait()))),
```

Comment:
```suggestion
                {asyncio.create_task(self.running.wait()), asyncio.create_task(self.update_triggered.wait())},
```

That's much easier to read

Reply:
let's see if black agrees